crates/types/src/joints.rs

```diff
@@ -60,6 +60,17 @@ impl Add for HeadJoints {
     }
 }
 
+impl Sub for HeadJoints {
+    type Output = HeadJoints;
+
+    fn sub(self, rhs: Self) -> Self::Output {
```

Comment:
Please don't abbreviate 

```suggestion
    fn sub(self, other: Self) -> Self::Output {
```

Also for the other occurrences.

Reply:
the `impl Add for Joints`, etc.. also uses `rhs`instead of `other`. Isn't `rhs` (although) abbreviated much more descriptive than `other`?

Reply:
We used `other` a lot already. If we decide on another name then we should also change the other occurrences. `right` would be okay for me.